content/en/docs/customization/deploying-containers/_index.md

```diff
@@ -4,24 +4,22 @@ date: 2023-01-03T17:24:56+05:30
 
-## Kanto-auto-deployer
+A standard set of containers is deployed through meta-leda, the manifests for which can be found in `meta-leda/meta-leda-components/recipes-sdv/eclipse-leda/kanto-containers/core` and `meta-leda/meta-leda-components/recipes-sdv/eclipse-leda/kanto-containers/example`.  
 
-This service has been implemented as a stopgap solution by the Leda team before the kanto-cm init-dir mechanism was implemented by the Kanto team. In-depth documentation on its operation and development can be found in the [Kanto Auto deployer](../../build/dev-and-maintenance/rust/kanto-auto-deployer). This runs as a one-shot service on boot and checks the directory defined in the BitBake variable `KANTO_MANIFESTS_DEV_DIR` in the distro config recipes. By default, the Leda quickstart images use `KANTO_MANIFESTS_DEV_DIR=/data/var/containers/manifests_dev"`. So, these manifest are again stored in the persistent **data**-partition and can be modified after the image has been deployed.
+The advantage of this service is that it can be restarted very quickly (`systemctl restart kanto-auto-deployer`), without having to restart the _whole_ container-management service. The implemented filewatcher (the `--daemon` flag) allows you to quickly create/edit  (or even `touch` them) container manifests on the device that would be on-the-fly deployed after saving. This allows for rapid testing when creating new container manifests.
 
-**Important: the [manifest template](../../build/dev-and-maintenance/rust/kanto-auto-deployer#manifest-structure) used for this service is similar to the one for the kanto-cm native mechanism, but is, in fact, different. Kanto-auto-deployer uses the kanto-cm internal state representation.**
+## Kanto Container Management init-dir
 
-A standard set of _dev_-containers is deployed with meta-leda, the manifests for which can be found in `meta-leda/meta-leda-components/recipes-sdv/eclipse-leda/kanto-containers/core_dev` and `meta-leda/meta-leda-components/recipes-sdv/eclipse-leda/kanto-containers/example_dev/`.
-
-The advantage of this service is that it can be restarted very quickly (`systemctl restart kanto-auto-deployer`), without having to restart the _whole_ container-management service. This allows for rapid testing when deploying a new container. Due to the similarity of both manifests, the kanto-auto-deployer-style manifest can later be easily migrated to the kanto ini-dir style-one.
+This feature of Kanto Container Managament by the Kanto Teams is currently under development and hence unstable. That is why it is not used as of the test-0.0.6 release of Leda Distro and 0.1.0-M1 release of meta-leda.
```

Comment:
I wouldn't say it's "unstable", but rather does not fit to our specific "Leda Quickstart"-use cases.